cmd/run.go

```diff
@@ -88,17 +94,28 @@ func runCommand(c *cli.Context) error {
 		return errors.New("wrong format for test case name, should be: `testplan/testcase`")
 	}
 
-	// Now that we've verified that the test plan and the test case exist, build
-	// the testplan.
-	buildIn, err := parseBuildInput(c)
-	if err != nil {
-		return fmt.Errorf("error while parsing build input: %w", err)
+	if noBuild && artifactPath == "" {
+		return errors.New("artifact-path is required when skipping the build")
 	}
 
-	// Trigger the build job.
-	buildOut, err := _engine.DoBuild(comp[0], builderId, buildIn)
-	if err != nil {
-		return fmt.Errorf("error while building test plan: %w", err)
+	if !noBuild && artifactPath != "" {
+		logging.S().Warn("artifact-path will be ignored, as we're performing a build")
+	}
```

Comment:
What about replacing both of these with "use-build"?

Reply:
👍 That simplifies things.